### PR TITLE
Add Invoice Types

### PR DESCRIPTION
**Pull request by mroig**
Added documentation of invoice types.

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_invoice_types/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_invoice_types/

File: docs/facturacion/tipos_facturas.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+Tal com s'explica a les factures Anul·ladora amb rectificadora, les factures
+Rectificadores i les Anul·ladores amb rectificadora sempre van aparellades.
+
+La funció de la parella és la de compensar una factura que s'ha emès amb dades
+incorrectes i facturar els valors que s'haurien d'haver facturat. Un cop hem
+compensat la factura original amb l'Anul·ladora amb Rectificadora, la factura
+Rectificadora ens servirà per facturar els valors correctes.
+
+**NOTA: A partir del 31/07/17 aquestes factures ja no es fan servir per enviar les
```
Comment (ecarreras) [inline docs/facturacion/tipos_facturas.md]: Pots afegir un `!!!note` i quedaria més ben presentat

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_invoice_types/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_invoice_types/

**Pull request merged**